frontend/src/components/atoms/SnackBarContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import SnackBarCloseButton from './SnackBarCloseButton';
+import { theme } from '../../styling/theme';
+import SnackBarLoader from './SnackBarLoader';
+
+interface ISnackBarProps {
+  content: string;
+  good: boolean;
+  className?: string;
+  speed?: string;
+  clicker?: () => void;
+}
+
+const LoaderSpeed = (speed = '6s') => {
+  if (speed === 'fast') {
+    return '4s';
+  } else if (speed === 'slow') {
+    return '8s';
+  } else {
+    return '6s';
+  }
+};
+
+const SnackBarContainer: React.FC<ISnackBarProps> = ({
+  content,
+  good,
+  className,
+  speed,
+  clicker,
+}) => {
+  return (
+    <div className={className}>
+      <p>{content}</p>
+      <SnackBarCloseButton onClick={clicker}>x</SnackBarCloseButton>
```

Comment:
Clicker should probably be `snackBarCloseHandler` or similar. And if it's a callback from an `onClick` on a button, it shouldn't be `() => void`, it should be `React.MouseEventHandler<HTMLButtonElement>`, i.e.

```tsx
interface IBlablablaProps {
  // ...
  onCloseHandler: React.MouseEventHandler<HTMLButtonElement>;
}
```

Reply:
Or if it is on the `div`, change `HTMLButtonElement` to `HTMLDivElement` etc.